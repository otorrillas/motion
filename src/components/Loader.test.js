import React from 'react';
import renderer from 'react-test-renderer';

import Loader from './Loader';

describe('<Loader />', () => {
  const renderComponent = () => renderer.create(<Loader />);

  describe('@render', () => {
    it('default', () => {
      const tree = renderComponent();
      expect(tree).toMatchSnapshot();
    });
  });
});
