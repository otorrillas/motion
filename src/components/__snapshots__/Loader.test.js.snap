// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> @render default 1`] = `
<div>
  Loading...
</div>
`;
